daffodil-test/src/test/scala/org/apache/daffodil/section06/entities/TestEntities.scala

```diff
@@ -92,16 +92,23 @@ class TestEntities {
   @Test def test_whitespace_09() { runner_01.runOneTest("whitespace_09") }
   @Test def test_whitespace_10() { runner_01.runOneTest("whitespace_10") }
 
-  // JIRA DFDL-1475 - Broken currently - test was wrong. Should expect error. Test changed, but diagnostic is not acceptable.
-  // @Test def test_emptyStringEntityTermInExpression_01() { runner_01.runOneTest("emptyStringEntityTermInExpression_01") }
+  // DAFFODIL-1475
+  @Test def test_emptyStringEntityTermInExpression_01() { runner_01.runOneTest("emptyStringEntityTermInExpression_01") }
   @Test def test_emptyStringEntityTermInExpression_02() { runner_01.runOneTest("emptyStringEntityTermInExpression_02") }
+  @Test def test_emptyStringEntityTermInExpressionDelimited_01() { runner_01.runOneTest("emptyStringEntityTermInExpressionDelimited_01") }
+  @Test def test_emptyStringEntityTermInComplex_01() { runner_01.runOneTest("emptyStringEntityTermInComplex_01") }
+  @Test def test_emptyStringEntityTermInComplex_02() { runner_01.runOneTest("emptyStringEntityTermInComplex_02") }
+
+  @Test def test_emptyStringEntityInitiator_01() { runner_01.runOneTest("emptyStringEntityInitiator_01") }
+  @Test def test_emptyStringEntityInitiator_02() { runner_01.runOneTest("emptyStringEntityInitiator_02") }
+  @Test def test_emptyStringEntityInitiator_03() { runner_01.runOneTest("emptyStringEntityInitiator_03") }
 
   @Test def test_entity_fail_01() { runnerEntity.runOneTest("entity_fail_01") }
   @Test def test_entity_fail_02() { runnerEntity.runOneTest("entity_fail_02") }
 
-  // Regression - we used to just reject %ES; in terminators. Now we accept it, but it doesn't work
-  // right. JIRA DFDL-1477
-  // @Test def test_entity_fail_03() { runnerEntity.runOneTest("entity_fail_03") }
+  // DAFFODIL-1477
+  @Test def test_entity_fail_03a() { runnerEntity.runOneTest("entity_fail_03a") }
+  @Test def test_entity_fail_03b() { runnerEntity.runOneTest("entity_fail_03b") }
```

Comment:
With this fix, we need to modify schemas for mil-std-2045, which I think uses this, but especially the FOUO schemas which make heavy use of this technique. At which point those schemas become dependent on this fix, but ultimately, without this fix they're incorrect and fragile.